spec/spec_helper.rb

```diff
@@ -0,0 +1,9 @@
+# encoding: utf-8
+
+require 'aws-sdk'
+require 'logstash/devutils/rspec/spec_helper'
+require 'logstash/outputs/sqs'
+require_relative 'supports/helpers'
+
+require 'logstash/logging/logger'
+LogStash::Logging::Logger::configure_logging('debug') if ENV['DEBUG']
```

Comment:
If this is desirable by default then maybe it should be moved to `logstash/devutils/rspec/spec_helper`?